src/main/java/com/google/devtools/build/lib/remote/RemoteCache.java

```diff
@@ -764,10 +758,11 @@ private ActionResultMetadata parseActionResultMetadata(
 
     ImmutableMap.Builder<Path, FileMetadata> files = ImmutableMap.builder();
     for (OutputFile outputFile : actionResult.getOutputFilesList()) {
+      Path path = remotePathResolver.resolveLocalPath(outputFile.getPath());
```

Comment:
opt: call this `localPath` maybe? (also above). I don't think it's much more descriptive, but given that `RemotePathResolver` seems to make the local/remote path distinction important, it looks like an improvement.

Reply:
Done.